ToolBox/PsychSignalDataConverter/PsychSignalDataConverterProgram.cs

```diff
@@ -0,0 +1,45 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System.IO;
+using QuantConnect.Logging;
+using DataConverter = QuantConnect.ToolBox.PsychSignalDataConverter.PsychSignalDataConverter;
+
+namespace QuantConnect.ToolBox.PsychSignalDataConverter
+{
+    public static class PsychSignalDataConverterProgram
+    {
+        /// <summary>
+        /// Entry point for ToolBox application PsychSignalDataConverter
+        /// </summary>
+        /// <param name="dataAbsolutePath"></param>
+        /// <param name="securityType"></param>
+        /// <param name="market"></param>
+        public static void PsychSignalDataConverter(string dataAbsolutePath, string securityType, string market)
+        {
+            Directory.CreateDirectory(Path.Combine(Globals.DataFolder, "equity", market, "alternative", "psychsignal"));
+
+            if (securityType != "equity")
```

Comment:
If equity is the only supported value then why accept it as an argument?

Reply:
Futures data is present in the data set, but I haven't got around to dealing with them yet as we're only after equity data at the current moment